pkgs/applications/networking/instant-messengers/twinkle/default.nix

```diff
@@ -15,20 +15,20 @@
 , alsa-lib
 , speex
 , ilbc
-, fetchurl
 , mkDerivation
 , bcg729
 }:
 
 mkDerivation rec {
   pname = "twinkle";
-  version = "1.10.2";
+  version = "unstable-2021-02-06";
 
   src = fetchFromGitHub {
-    repo = pname;
     owner = "LubosD";
-    rev = "v${version}";
-    sha256 = "0s0gi03xwvzp02ah4q6j33r9jx9nbayr6dxlg2ck9pwbay1nq1hx";
+    repo = "twinkle";
+    rev = "2301b66a3f54b266675415d261985488d86e9e4c";
+    sha256 = "xSwcaj1Hm62iL7C/AxqjVR07VEae8gDgYdr2EWmCoOM=";
+    fetchSubmodules = false;
```

Comment:
```suggestion
```
that is the default. Not required to repeat it.

Reply:
https://github.com/seppeljordan/nix-prefetch-github/pull/42/files